### ISSUE TITLE
Disable new database prompt when importing sql file

### ISSUE THREAD

**Issue by mtissington** [sqlitebrowser/sqlitebrowser#1680]
When importing an sql file into an existing database we are always asked if we want to import into a new database.

Would be good if we could disable this prompt.
![annotation 2018-12-21 145542](https://user-images.githubusercontent.com/6769116/50348448-a8331580-0530-11e9-8df0-75ba3c9f8346.jpg)

**Comment by justinclift** [comment]
Maybe something like a:

* `[ ] Don't ask again`

... option on the Yes/No line?

Would probably also need some place in the Preferences area to toggle/reset it back on, for when people need to use a different answer at some point. :smile:

**Comment by mtissington** [comment]
Yes, something in Preferences is required to re-enable the question.
This has nothing to do with DB Browser but is an example of an array of Preferences/Warnings that can be switched off when they are first displayed and if need be re-enabled thru Preferences.

![defaults](https://user-images.githubusercontent.com/6769116/50365842-590bd580-056e-11e9-97e0-85c2d7d86118.jpg)

**Comment by mtissington** [comment]
If I open Word or Excel, or CSV and again I have an existing database open - the probability is high that I want to import into the already open database. I would change you last question to the following...

In what scenario would you open a database, go to import ANY file and want that in ANOTHER database?
